lib/explorer/backend/lazy_frame.ex

```diff
@@ -73,10 +74,21 @@ defmodule Explorer.Backend.LazyFrame do
   @impl true
   def pull(df, column) do
     dtype_for_column = df.dtypes[column]
+    series_backend = get_series_backend(df.data.original_data)
+
     data = LazySeries.new(:column, [column], dtype_for_column)
```

Comment:
And we should probably pass the backend as a required argument here. We need to make sure we always pass it forward, otherwise something like `re_named_captures(s.foo <> s.bar, ~S"...")` won't work if we forget to pass the backend inside? Would this be too large of a change?

Reply:
I'm passing down based on the `args` - see the [`new/3` function at LazySeries](https://github.com/elixir-explorer/explorer/pull/896/files#diff-f91b78e498841f6f17ff64be6801aab96764b8d998a5d69c6b397da4d20de35fR208) -, which will mostly contain a lazy series. The only exceptions are `from_list/2` and `from_binary/2`. I honestly don't know how to solve this, because there is no point of reference to get the backend. I was thinking about getting the default backend for them. WDYT?

Reply:
Using the default in c708f46